src/js/components/home/ProgressSection.js

```diff
@@ -0,0 +1,144 @@
+    const { hasBeenFetched } = this.props.progress;
+    if (!hasBeenFetched) {
+      progressStat = <InlineLoader/>
+    } else {
+      progressStat = (
+        <span>
+          {data.value}/<span>{data.total}</span>
+        </span>
+      );
+    }
+
+    return (
+      <div className="col-md-4">
+        <div className={`progress-circle progress-circle-${color}`} data-goal={data.percentage} data-progress="0">
+          <div className="circle">
+              <div className="full progress-circle-slice">
+                  <div className="progress-circle-fill" />
+              </div>
+              <div className="progress-circle-slice">
+                  <div className="progress-circle-fill" />
+                  <div className="progress-circle-fill progress-circle-bar" />
+              </div>
+          </div>
+          <div className="progress-circle-overlay">
+            <span className="progress-circle-percent">
+              {progressStat}
+              <p className="progress-circle-caption">{goal}</p>
+            </span>
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  render() {
+    const { progressData } = this.props.progress;
+    const countriesData = this.getProgressValueAndPercentage(progressData, 'countries', 2);
+    const schoolsData = this.getProgressValueAndPercentage(progressData, 'schools', 100);
+    const studentsData = this.getProgressValueAndPercentage(progressData, 'students', 1000);
+
+    return (
+      <section id="how" className="goals section">
+        <div className="goals-overlay" />
+        <div className="container">
+          <div className="row">
+            <div className="col-lg-12 text-xs-center">
+              <h4 className="section-heading">Our Goals for 2017</h4>
+              <h3 className="section-subheading text-muted" />
+            </div>
+          </div>
+          <div className="row" id="progress">
+            <div className="col-xs-12">
+              <div className="row">
+                {this.generateProgressCircle(countriesData, 'countries', 'orange')}
+                {this.generateProgressCircle(schoolsData, 'schools reached', 'green')}
+                {this.generateProgressCircle(studentsData, 'students enrolled', 'blue')}
+              </div>
+            </div>
+          </div>
+          <div className="row">
+            <div className="col-xs-12">
+              <div className="text-sm-right text-xs-center btn-mission">
+                <Link to="/mission" className="btn btn-primary">See why these numbers are important to us</Link>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    )
+  }
+}
+
+ProgressSection.propTypes = {
+  getProgresses: PropTypes.func,
+  progress: PropTypes.object
+};
+
+const mapStateToProps = (state) => {
+  return {
+    progress: state.progress
+  };
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return bindActionCreators({
+    getProgresses: getProgresses
```

Comment:
Not necessary :), but we could have just one `getProgresses` and es6 will be fine :)